docs/output.md

```diff
@@ -482,17 +440,55 @@ The file `[TUMORSAMPLE].cnvs.txt` contains all segments predicted by ASCAT, both
 
 For further reading and documentation see the [ASCAT manual](https://www.crick.ac.uk/research/labs/peter-van-loo/software).
 
+#### CNVKit
+
+[CNVKit](https://cnvkit.readthedocs.io/en/stable/) is a toolkit to infer and visualize copy number from high-throughput DNA sequencing data. It is designed for use with hybrid capture, including both whole-exome and custom target panels, and short-read sequencing platforms such as Illumina and Ion Torrent.
+
+For normal-only or tumor-only samples:
+
+**Output directory: `results/variantcalling/<sample>/cnvkit`**
+
+- `multi_intervals.antitarget.bed`
+- `<sample>.antitargetcoverage.cnn`
+- `<sample>.targetcoverage.cnn`
+- `reference.cnn`
+- `<sample>-diagram.pdf`
+- `<sample>-scatter.png`
+- `<sample>.bintest.cns`
+- `multi_intervals.target.bed`
+- `<sample>.cnr`
+- `<sample>.cns`
+- `<sample>.call.cns`
+
+For tumor/normal pairs:
+
+**Output directory: `results/variantcalling/<tumorsample_vs_normalsample>/cnvkit`**
+
+multi_intervals.antitarget.bed
+test.paired_end.recalibrated.sorted.antitargetcoverage.cnn
```

Comment:
In this list of files, should `test` and `test2` perhaps be `<normal_sample>` and `<tumor_sample>`?